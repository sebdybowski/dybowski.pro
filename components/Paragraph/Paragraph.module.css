.Paragraph {
  font-size: 1.5rem;
  margin: 1.5rem 0;
}