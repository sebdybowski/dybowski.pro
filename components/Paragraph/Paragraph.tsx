import React, { FC } from 'react';

import styles from './Paragraph.module.scss';

type ParagraphProps = { children: string };

const Paragraph: FC<ParagraphProps> = ({ children }) => (
  <p className={styles.Paragraph} data-testid="Paragraph">
    {children}
  </p>
);

export default Paragraph;
