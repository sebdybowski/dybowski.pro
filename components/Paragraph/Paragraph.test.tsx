import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Paragraph from './Paragraph';

describe('<Paragraph />', () => {
  test('it should mount', () => {
    render(<Paragraph />);
    
    const paragraph = screen.getByTestId('Paragraph');

    expect(paragraph).toBeInTheDocument();
  });
});