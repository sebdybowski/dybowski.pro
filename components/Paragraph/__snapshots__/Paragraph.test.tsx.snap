// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph /> it should mount 1`] = `
<DocumentFragment>
  <p
    class="Paragraph"
    data-testid="Paragraph"
  >
    Content
  </p>
</DocumentFragment>
`;
